APIContract.md

```diff
@@ -0,0 +1,229 @@
+                  "user_id": 1,
+                  "name": "Lassy",
+                  "age": 5,
+                  "breed": "Boxer",
+                  "description": "Very nice and lovable",
+                  "gender": "F",
+                  "fix": false,
+                  "house trained": true,
+                  "good with kids": true,
+                  "good with other pets": true,
+                  "photo_url_1": "URL Path",
+                  "photo_url_2": "URL Path",
+                  "photo_url_3": "URL Path",
+                }
+            }
+          ]
+        },
+        "favorates": {
+          "pets": {
+            "data": [
+              {
+                "id": "6",
+                  "type": "pet",
+                  "attributes": {
+                    "type": "cat",
+                    "user_id": 4,
+                    "name": "Purrfect",
+                    "age": 2,
+                    "breed": "",
+                    "description": "nice and friendly",
+                    "gender": "F",
+                    "fix": true,
+                    "house trained": true,
+                    "good with kids": true,
+                    "good with other pets": true,
+                    "photo_url_1": "URL Path",
+                    "photo_url_2": "URL Path",
+                    "photo_url_3": "URL Path",
+                  }
+              },
+              {
+                "id": "4",
+                  "type": "pet",
+                  "attributes": {
+                    "user_id": 1,
+                    "name": "Lassy",
+                    "age": 5,
+                    "breed": "Boxer",
+                    "description": "Very nice and lovable",
+                    "gender": "F",
+                    "fix": false,
+                    "house trained": true,
+                    "good with kids": true,
+                    "good with other pets": true,
+                    "photo_url_1": "URL Path",
+                    "photo_url_2": "URL Path",
+                    "photo_url_3": "URL Path",
+                  }
+              }
+            ]
+          }
+        },
+        }
+      }
+    }
+  }
+  ```
+
+## Create New User
+`POST /api/v1/users`
+  ```json
+  {
+    "data": {
+      "id": "1",
+      "type": "User",
+      "attributes": {
+        "username": "My Name",
+        "email": "my@email.com"
+      }
+    }
+  }
+  ```
+
+## Create New Pet
+`POST /api/v1/pets`
+  ```json
+  {
+    "data": {
+      "id": "1",
+      "type": "pet",
+      "attributes": {
+        "user_id": 9,
+        "name": "Sport",
+        "age": 0,
+        "breed": "Lab",
+        "description": "Good Boy",
+        "gender": "M",
+        "fix": false,
+        "house trained": false,
+        "good with kids": true,
+        "good with other pets": true,
+        "photo_url_1": "URL Path",
+        "photo_url_2": "URL Path",
+        "photo_url_3": "URL Path",
+      }
+    }
+  }
+  ```
+
+## Delete a Booking
```

Comment:
Need to change the wording on this one from 'booking' to 'pet'

Reply:
Done! thanks